kudago-service/src/main/java/com/example/controller/CategoryRestController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.controller;
+
+import com.example.aspect.LogExecutionTime;
+import com.example.controller.payload.CategoryPayload;
+import com.example.entity.Category;
+import com.example.service.CategoryService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.MessageSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.NoSuchElementException;
+
+@LogExecutionTime
+@RestController
+@RequestMapping("/api/v1/places/categories")
+@RequiredArgsConstructor
+public class CategoryRestController {
+
+    private final CategoryService categoryService;
+    private final MessageSource messageSource;
+
+    @GetMapping
+    public ResponseEntity<List<Category>> getAllCategories() {
```

Comment:
Для Ok не надо обрачивать в ResponseEntity ```suggestion
    public List<Category> getAllCategories() {
```